cmd/eval.go

```diff
@@ -92,20 +93,21 @@ Example: combo eval -r REPLACE_ME=1,2,3 path/to/file
 			)
 
 			templateBuilder, err := template.NewBuilder(templateFile, combinations)
+
 			if err != nil {
 				return fmt.Errorf("failed to construct builder: %w", err)
 			}
 
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
-
 			combinedTemplateManifests, err := templateBuilder.Build(ctx)
 			if err != nil {
 				return fmt.Errorf("failed to build manifests with combinations: %w", err)
 			}
 
 			combinedTemplate := "---\n" + strings.Join(combinedTemplateManifests, "\n---\n")
 
+			// validate created file
```

Comment:
nit: I think we can remove this comment as the function name should be self-explanatory

Reply:
Got it.